scripts/bench-kungfu.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/sh
+set -e
+
+cd $(dirname $0)
+
+KUNGFU_SRC=$(pwd)/KungFu
+
+install_kungfu() {
+    local KUNGFU_GIT_URL=git@github.com:lsds/KungFu.git
+    if [ ! -d ${KUNGFU_SRC} ]; then
+        git clone ${KUNGFU_GIT_URL} ${KUNGFU_SRC} -b docker
+    fi
+    cd ${KUNGFU_SRC}
+    ./scripts/azure/gpu-machine/install-golang1.11.sh
+    export PATH=$HOME/local/go/bin:$PATH
+    ./scripts/go-install.sh --build-gtest
+    pip install --no-index -U .
+    cd -
+}
+
+KUNGFU_PRUN=${KUNGFU_SRC}/bin/kungfu-prun
+if [ ! -f $KUNGFU_PRUN ]; then
+    install_kungfu
+fi
+
+np=4
+$KUNGFU_PRUN \
+    -np=$np \
+    -timeout=10m \
+    python \
+    tf_cnn_benchmarks/tf_cnn_benchmarks.py \
+    --num_gpus=$np \
```

Comment:
The $np are both passed to KUNGFU_PRUN and tf_cnn_benchmarks. Why we need to wrap the python program within a KUNGFU_PRUN here?

Reply:
If we are using kung-prun outside, the --num_gpus shall be set to 1 like Horovod, right?

Reply:
@lgarithm